LoopKit/CarbKit/CarbMath.swift

```diff
@@ -9,7 +9,7 @@
 import Foundation
 import HealthKit
 
-
+// dm61 TODO(?): update protocol to include percentRateAtPercentTime, refactor all carb absorption models to conform to the updated protocol, move protocol and carb absorption models into separate files, unify references to absorption model, make it easy to select the active carb absorption model (could even make the model user selectable, similar to what we do with insulin models?)
```

Comment:
The code-level model selection could be as simple as `typealias AbsorptionModel = LinearAbsorption` as default, with alternatives such as `typealias AbsorptionModel = PiecewiseLinearAbsorption` commented out. Would this be acceptable?

Reply:
Look at https://github.com/LoopKit/Loop/blob/dev/Common/FeatureFlags.swift, to see how this could be done as a feature flag.

Reply:
Since this is a LoopKit level patch, providing a new absorption model argument to the CarbStore would probably be the proper approach, then we can pass the correct one in based on the Loop feature flag.

Reply:
Awesome, thanks for the guidance and the example - exactly what I was looking for!